ogr/services/github.py

```diff
@@ -528,6 +539,10 @@ class GithubUser(BaseGitUser):
     def __init__(self, service: GithubService) -> None:
         super().__init__(service=service)
 
+    def __str__(self) -> str:
+        return (
+            f"Github-User"
```

Comment:
would be nice to see the login

Reply:
Yes I will make the changes. Thanks for feedback !